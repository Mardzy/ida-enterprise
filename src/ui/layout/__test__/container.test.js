import Container from "../container";

describe("The Container component", () => {

  it("sets bottom margin when the whitespaceAfter prop is passed", () => {
    const wrapper = mount(<Container whitespaceAfter={2}>Lorem ipsum dolor</Container>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});