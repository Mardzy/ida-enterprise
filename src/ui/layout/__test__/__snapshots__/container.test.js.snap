// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Container component sets bottom margin when the whitespaceAfter prop is passed 1`] = `
.c0 {
  margin-bottom: 2rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<Styled(Component)
  whitespaceAfter={2}
>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "
  margin-bottom: ",
            [Function],
            ";
  
  &:last-child {
    margin-bottom: 0;
  }
",
          ],
        },
        "displayName": "Styled(Component)",
        "foldedComponentIds": Array [],
        "propTypes": Object {
          "whitespaceAfter": [Function],
        },
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": [Function],
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
    whitespaceAfter={2}
  >
    <Component
      className="c0"
      whitespaceAfter={2}
    >
      <section
        className="c0"
      >
        Lorem ipsum dolor
      </section>
    </Component>
  </StyledComponent>
</Styled(Component)>
`;
